Dockerfile

```diff
@@ -18,11 +18,11 @@ RUN pip install -r /home/murphy/requirements.txt --src /usr/local/src
 # install library
 ADD ./r3 /home/murphy/r3
 ADD ./swagger_ui /home/murphy/swagger_ui
-ADD ./main.py /home/murphy/main.py
 ADD ./setup.py /home/murphy/setup.py
-ADD ./src /home/murphy/src
+ADD ./src /home/murphy/node_normalizer
```

Comment:
Hi @kshefchek , I think this needs to be `ADD ./node_normalizer /home/murphy/node_normalizer`
also we need `ADD ./config.json /home/murphy/config.json`

Reply:
done!